app/views/layouts/application.html.erb

```diff
@@ -4,16 +4,30 @@
     <title>Reddit Clone</title>
     <%= csrf_meta_tags %>
     <%= csp_meta_tag %>
-
     <%= stylesheet_link_tag 'application', media: 'all' %>
     <%= stylesheet_link_tag 'https://stackpath.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css' %>
     <script src="https://code.jquery.com/jquery-3.4.1.min.js" integrity="sha256-CSXorXvZcTkaix6Yvo6HppcZGetbYMGWSFlBw8HfCJo=" crossorigin="anonymous"></script>
     <%= javascript_pack_tag 'application' %>
   </head>
-
   <body>
     <%= render partial: "layouts/navbar" %>
     <%= yield %>
     <%= render partial: "layouts/footer" %>
+    <% if flash[:notice] %>
+      <script>
+        toastr.options = {
+            "closeButton": true
+        };
```

Comment:
why is this part duplicated? can we refactor it ?

Reply:
code refactored